Integrations/integration-Zscaler.yml

```diff
@@ -814,34 +859,38 @@ script:
       description: Category ID
       type: string
     - contextPath: Zscaler.Category.URL
-      description: List of IP addresses in the category
+      description: List of category URL addresses
+      type: string
     description: Adds IP address to given category
   - name: zscaler-category-remove-url
     arguments:
     - name: category-id
       required: true
-      description: Category ID to remove URL from, e.g. RADIO_STATIONS
+      description: Category ID to remove URL from, e.g. RADIO_STATIONS (Could be retreived
+        by running the command zscaler-get-categories)
     - name: url
       required: true
       description: URL address to remove. Comma separated values supported, e.g. pandora.com,spotify.com
     outputs:
-    - contextPath: Zscaler.Category.CustomCategory
+    - contextPath: ' Zscaler.Category.CustomCategory'
       description: True if category is custom, else false.
       type: boolean
     - contextPath: Zscaler.Category.Description
       description: Category description
       type: string
     - contextPath: Zscaler.Category.ID
       description: Category ID
-      type: string
+      type: number
```

Comment:
in some place it is string, here it number. So category ID is a number or string?